terraform/environments/electronic-monitoring-data/modules/dms/RDS_MSSQLServer_DB_Scripts/D_Comments_V2_Insert_Into_Select.sql

```diff
@@ -0,0 +1,27 @@
+SET IDENTITY_INSERT g4s_cap_dw.dbo.D_Comments_V2 ON
+;
+
+INSERT INTO g4s_cap_dw.dbo.D_Comments_V2 (CommentSID, VisitID, ActivityID, Comments, CommentType)
+ SELECT CommentSID, VisitID, ActivityID, trim(replace(Comments, char(141), '')) AS Comments, CommentType
+  FROM g4s_cap_dw.dbo.D_Comments
+;
+
+SET IDENTITY_INSERT g4s_cap_dw.dbo.D_Comments_V2 OFF;
+
+
+-- TESTING QUERIES -- 
+--
+-- SELECT COUNT(*) FROM g4s_cap_dw.dbo.D_Comments_V2; -- 49695569
+-- SELECT COUNT(*) FROM g4s_cap_dw.dbo.D_Comments; -- 49695569
+--
+
+
+--
+-- SELECT 'D_Comments' AS TableName, Comments
+--   FROM g4s_cap_dw.dbo.D_Comments
+--  WHERE CommentSID = 26837791
+-- UNION
+-- SELECT 'D_Comments_V2' AS TableName, Comments
+--   FROM g4s_cap_dw.dbo.D_Comments_V2
+--  WHERE CommentSID = 26837791
+--
```

Comment:
Do we need these comments?

Reply:
Yes, These were post-transformation testing queries essential to confirm the transformation doing as expected.
I am wondering where I can push these if needs to be kept in the repo and visible !!